en/guide/mavlink_v1_compatibility_features.md

```diff
@@ -0,0 +1,97 @@
+# MAVLink v1.1 Compatibility and Features
+
+The MAVLink v1.1 release is the first release that requires a minimum set of messages to be supported to qualify as *MAVLink enabled*. 
+It will replaces a number of historically suboptimal messages with newer messages, while retaining full compatibility with v1.0.
+
+## Capability and Autopilot Version Discovery Mechanism
+
+The [AUTOPILOT_VERSION](../messages/common.md#AUTOPILOT_VERSION) message carries a bit field which encodes the generic MAVLink features this autopilot supports. 
+It should be sufficient to read off all relevant properties of the autopilot to communicate. 
+It should be sent every 0.1 Hz at least. 
+The GCS can configure itself according to the announced features and highlight if the user should update the firmware version.
+
+## Message Set Requirements
+
+The message set requirements should be interpreted as follows: If the autopilot sends any dimension contained in the messages below, it has to use these messages and not deprecated other messages to do so. 
```

Comment:
I don't think we need this line unless we really are using this to create a "standard" for verification.